sys/events.js

```diff
@@ -24,6 +24,17 @@ EventService.prototype.emitEvent = function(hyper, req) {
         return { status: 200 };
     }
     return P.try(function() {
+        // Change-propagation will set up the x-triggered-by header, indicating
+        // the event which caused the rerender. In case RESTBase is about to emit
+        // the same event, it will cause a rerender loop. So, log an error and skip
+        // the event.
+        var triggeredBy = req.headers && req.headers['x-triggered-by']
+            || hyper._rootReq && hyper._rootReq['x-triggered-by'];
+        var topic = self.options.topic;
+        if (triggeredBy && /transcludes/.test(triggeredBy)) {
+            topic = self.options.transcludes_topic;
```

Comment:
Since this is a new configuration option, we should either provide a default or fail on start-up if it hasn't been provided. Perhaps the former is more appropriate here.